rest-api-spec/src/main/resources/rest-api-spec/api/streams.logs_disable.json

```diff
@@ -0,0 +1,27 @@
+{
+  "streams.logs_disable": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/streams-logs-disable.html",
+      "description": "Disable the Logs Streams feature for this cluster"
+    },
+    "stability": "stable",
+    "visibility": "public",
```

Comment:
I think you want "visibility": "feature_flag" for these so that they don't get auto-added to our public spec yet.